etc/containerpilot.json

```diff
@@ -17,6 +17,14 @@
       "onChange": "python /usr/local/bin/manage.py on_change"
     }
   ],
+  "tasks": [
+    {
+      "name": "snapshot",
+      "command": "python /usr/local/bin/manage.py snapshot_task",
+      "frequency": 5,
```

Comment:
We're tolerant of fuzzy inputs. Our parsing for these defaults to seconds when we pass in an int. So this would be 5 seconds. That being said, as being intended as an example we should probably pass in `"5s"` here.